syntax/fetlang.vim

```diff
@@ -2,50 +2,45 @@ if exists('b:current_syntax')
 	finish
 endif
 
-" -- Keyword {{{1
-" --------------------------------------------------------------------------------------------------
-
-syntax match fetlangKeyword '\v\c<spank>'
-syntax match fetlangKeyword '\v\c<worship>'
-syntax match fetlangKeyword '\v\c<flog>'
-syntax match fetlangKeyword '\v\c<lick>'
-syntax match fetlangKeyword '\v\c<tickle>'
-syntax match fetlangKeyword '\v\c<moan>'
-syntax match fetlangKeyword '\v\c<scream>'
-syntax match fetlangKeyword '\v\c<torture>'
-syntax match fetlangKeyword '\v\c<tie up>'
-syntax match fetlangKeyword '\v\c<hogtie>'
-syntax match fetlangKeyword '\v\c<assert dominance over>'
-syntax match fetlangKeyword '\v\c<assert equality with>'
-syntax match fetlangKeyword '\v\c<demand obedience of>'
-syntax match fetlangKeyword '\v\c<obey>'
-syntax match fetlangKeyword '\v\c<serve>'
-
-highlight link fetlangKeyword Keyword
-
 " -- Function {{{1
 " --------------------------------------------------------------------------------------------------
 
-syntax match fetlangFunction '\v\c<make>'
-syntax match fetlangFunction '\v\c<have>'
-syntax match fetlangFunction '\v\c<bind>'
-syntax match fetlangFunction '\v\c<call safeword>'
+syntax match fetlangFunction '\v\c<spank>'
+syntax match fetlangFunction '\v\c<worship>'
+syntax match fetlangFunction '\v\c<flog>'
+syntax match fetlangFunction '\v\c<lick>'
+syntax match fetlangFunction '\v\c<tickle>'
+syntax match fetlangFunction '\v\c<moan>'
+syntax match fetlangFunction '\v\c<scream>'
+syntax match fetlangFunction '\v\c<torture>'
+syntax match fetlangFunction '\v\c<tie up>'
+syntax match fetlangFunction '\v\c<hogtie>'
+syntax match fetlangFunction '\v\c<assert dominance over>'
+syntax match fetlangFunction '\v\c<assert equality with>'
+syntax match fetlangFunction '\v\c<demand obedience of>'
+syntax match fetlangFunction '\v\c<obey>'
+syntax match fetlangFunction '\v\c<serve>'
 
 highlight link fetlangFunction Function
 
-" -- Repeat {{{1
+" -- Keyword {{{1
 " --------------------------------------------------------------------------------------------------
 
-syntax match fetlangRepeat '\v\c<while>'
-syntax match fetlangRepeat '\v\c<until>'
```

Comment:
`Repeat` category is special only for that kind of commands. Do you still think they should be under `Keyword`?

Reply:
Again, never used vimscript, but when I write C in vim, while and if seem to be the same color. You can go ahead and revert it.

Reply:
That's okay. If that's the case, i'll make adjustments.